internal/router/router.go

```diff
@@ -16,6 +17,8 @@ var Router TRouter
 // CreateServer initializes router and creates routes for each application
 func CreateServer(apps map[string]*app.App) (*fiber.App, error) {
 	r := fiber.New()
+	// todo: REMOVE DEBUG SETTING
+	r.Use(cors.New())
```

Comment:
In future, we should have options in config for cors. Let it be everywhere, for now. Remove comment about debug.